packages/calling/src/CallSettings/CallSettings.ts

```diff
@@ -17,9 +17,18 @@ import {CALL_SETTINGS_FILE} from './constants';
 import {UcmBackendConnector} from './UcmBackendConnector';
 
 /**
- * Call Settings class which implements the public ICallSettings interface
- * This class will select appropriate calling backends according to the user
- * entitlements.
+ * CallSettings class represents the CallSettings Client Module
+ * Purpose of this class is to provide the APIs to fetch and update the settings like CallWaiting, DND, CallForward, Voicemail etc. based on different calling backends.
+ * The appropriate calling backends is initialized according to the user entitlements while intsantiating the CallSettings Client.
+ *
+ * To access the APIs, instance of CallSettings Client is required.
```

Comment:
If we aren't going to make the users do a `new CallSettings()` routine, I think we need not mention anything about the class and directly talk about the instance itself. 

```suggestion
 * The purpose of this callSettings instance is to provide the APIs to fetch and update the settings like CallWaiting, DND, CallForward, Voicemail, etc., based on different calling backends.
 * The appropriate calling backends is initialized according to the user entitlements while instantiating the CallSettings Client.
 *
 * To access the APIs, instance of CallSettings Client is required.
```

Also, there's a minor typo in the word `instantiating`. Kindly check

Reply:
Taken care